### PR TITLE
compile atf fip image

### PR DESCRIPTION
**Pull request by muvarov**
fip firmware image includes atf, optee and uboot.

Signed-off-by: Maxim Uvarov <maxim.uvarov@linaro.org>

### CODE REVIEW

**Comment by muvarov** [comment]
> Wouldn't it make sense to change the QEMU runqemu commands as well and add OP-TEE/TFA/U-boot booting as standard?

yep. I think that is also good idea.

**Comment by muvarov** [comment]
@cpriouzeau please review stm changes. I guess default boot method is emmc?

File: meta-ledge-bsp/recipes-bsp/arm-trusted-firmware/arm-trusted-firmware-ledge_git.bb [hunk 1]
```diff
@@ -14,7 +14,7 @@ PACKAGE_ARCH = "${MACHINE_ARCH}"
 PV = "2.1"
```
Comment (cpriouzeau) [inline meta-ledge-bsp/recipes-bsp/arm-trusted-firmware/arm-trusted-firmware-ledge_git.bb:14]: Please change the version, with this SHA1 you are switched to version v2.3

File: meta-ledge-bsp/recipes-bsp/arm-trusted-firmware/arm-trusted-firmware-ledge_git.bb [hunk 2]
```diff
@@ -40,10 +42,14 @@ EXTRA_OEMAKE = ' CROSS_COMPILE=${TARGET_PREFIX} '
 EXTRA_OEMAKE += ' PLAT=${TF_A_PLATFORM} '
 EXTRA_OEMAKE_append_armv7a = ' ARCH=aarch32 ARM_ARCH_MAJOR=7 '
 EXTRA_OEMAKE_append_armv7ve = ' ARCH=aarch32 ARM_ARCH_MAJOR=7 '
-EXTRA_OEMAKE_append_ledge-stm32mp157c-dk2 = "AARCH32_SP=optee"
+EXTRA_OEMAKE_append_ledge-stm32mp157c-dk2 = "AARCH32_SP=optee STM32MP_EMMC=1"
```
Comment (cpriouzeau) [inline meta-ledge-bsp/recipes-bsp/arm-trusted-firmware/arm-trusted-firmware-ledge_git.bb:45]: For SDCARD, you must specify "STM32MP_SDMMC=1" and here it's only for emmc

Comment (muvarov) [reply meta-ledge-bsp/recipes-bsp/arm-trusted-firmware/arm-trusted-firmware-ledge_git.bb:45]: @cpriouzeau specify both or only SDMMC=1?

Comment (cpriouzeau) [reply meta-ledge-bsp/recipes-bsp/arm-trusted-firmware/arm-trusted-firmware-ledge_git.bb:45]: @muvarov you can specify the both.

**Pull request merged**